pyproject.toml

```
@@ -11,21 +11,23 @@ description = "MQT Predictor - A MQT tool for Determining Good Quantum Circuit C
 readme = "README.md"
 authors = [
     { name = "Nils Quetschlich", email = "nils.quetschlich@tum.de" },
-    { name = "Lukas Burgholzer", email = "lukas.burgholzer@jku.at"},
+    { name = "Lukas Burgholzer", email = "lukas.burgholzer@tum.de"},
 ]
 keywords = ["MQT",  "quantum computing", "compilation", "machine learning", "prediction"]
 license = { file = "LICENSE" }
 requires-python = ">=3.10"
 dynamic = ["version"]
 
 dependencies = [
-    "mqt.bench @ git+https://github.com/cda-tum/mqt-bench.git",
+    "mqt.bench>=1.1.1, <1.1.2",
```

That seems like a very tight range to me.
According to semantic versioning, it should be fine to do
```suggestion
    "mqt.bench>=1.1.1, <1.2.0",
```